util/libcrypto.num

```diff
@@ -4647,8 +4647,8 @@ OSSL_CMP_ITAV_get0_value                4763	3_0_0	EXIST::FUNCTION:CMP
 OSSL_CMP_ITAV_push0_stack_item          4764	3_0_0	EXIST::FUNCTION:CMP
 OSSL_CMP_ITAV_free                      4765	3_0_0	EXIST::FUNCTION:CMP
 OSSL_CMP_MSG_free                       4766	3_0_0	EXIST::FUNCTION:CMP
-OSSL_CMP_PKISI_free                     4767	3_0_0	EXIST::FUNCTION:CMP
-OSSL_CMP_MSG_dup                        4768	3_0_0	EXIST::FUNCTION:CMP
+OSSL_CMP_PKISI_free                     4767	3_0_0	NOEXIST::FUNCTION:CMP
+OSSL_CMP_MSG_dup                        4768	3_0_0	NOEXIST::FUNCTION:CMP
```

Comment:
Will this be removed?

Reply:
Argh, the design of `libcrypto.num` always causes trouble.
The problem is that, as you suggested, we un-exported these two functions, 
so `make ordinals`, unless corrected manually, reports such functions as removed.
After the correction, which we had forgotten this time but which I've just done,
the numbers for the functions in all following lines need to be shifted :-(

Reply:
OK, thanks.